projects/storefrontlib/src/events/events.module.ts

```diff
@@ -1,9 +1,15 @@
 import { NgModule } from '@angular/core';
 import { CartPageEventModule } from './cart/cart-page-event.module';
+import { NavigationEventModule } from './navigation/navigation-event.module';
 import { PageEventModule } from './page/page-event.module';
 import { ProductPageEventModule } from './product/product-page-event.module';
 
 @NgModule({
-  imports: [CartPageEventModule, PageEventModule, ProductPageEventModule],
+  imports: [
+    NavigationEventModule,
```

Comment:
As this module is deprecated already on develop, I'm wondering, if we should introduce changes to it.

Reply:
Added to `projects/storefrontlib/src/recipes/storefront-foundation.module.ts` instead

Reply:
Its also deprecated ;-)

Reply:
Yes, I realized that later. Move to _spartacus feature module_